roles/container_runtime/templates/docker_storage_setup.j2

```diff
@@ -2,11 +2,15 @@
 # /usr/lib/docker-storage-setup/docker-storage-setup.
 #
 # For more details refer to "man docker-storage-setup"
+{% if container_runtime_docker_storage_setup_device is defined and container_runtime_docker_storage_setup_device != '' %}
 DEVS={{ container_runtime_docker_storage_setup_device }}
 VG={{ docker_storage_setup_options.vg }}
 DATA_SIZE={{ docker_storage_setup_options.data_size }}
 STORAGE_DRIVER="{{ docker_storage_setup_options.storage_driver }}"
 CONTAINER_ROOT_LV_NAME="{{ docker_storage_setup_options.root_lv_name }}"
 CONTAINER_ROOT_LV_SIZE="{{ docker_storage_setup_options.root_lv_size }}"
 CONTAINER_ROOT_LV_MOUNT_PATH="{{ docker_storage_setup_options.root_lv_mount_path }}"
+{% else %}
+STORAGE_DRIVER="{{ docker_storage_setup_options.storage_driver }}"
```

Comment:
Doesn't storage driver have to be empty here to tell docker storage setup to do nothing?  Or will it do the correct thing otherwise?

Reply:
I'm not 100% sure.  If you'd prefer to leave out the `STORAGE_DRIVER` option we can try.

I tested with a single drive in AWS and set the following variables:
```
container_runtime_docker_storage_type: overlay2
container_runtime_docker_storage_setup_device: ''
```

This produced the following docker-storage-setup file:
```
STORAGE_DRIVER="overlay2"
EXTRA_STORAGE_OPTIONS="--storage-opt overlay2.override_kernel_check=true  --graph=/var/lib/docker"
```
Here is the docker info output:
``` 
[root@ip-172-31-58-161 ~]# docker info | grep overlay
Storage Driver: overlay2
 Network: host bridge null overlay
```